custom-service-adapter/README.md

```diff
@@ -0,0 +1,19 @@
+# Custom Service Adatper with Knot.x
+
+In this tutorial we create a page: [http://localhost:8092/html/books.html](http://localhost:8092/html/books.html) books from database.
+Database access is not available via WebAPI thus we have 2 options:
+
+1. integrate with data source implementing WebAPI layer
```

Comment:
This is not what I had in mind. First option is to integrate only via frontend integration (e.g. AJAX call).